src/test/kotlin/Test.kt

```diff
@@ -0,0 +1,20 @@
+import org.junit.jupiter.api.Assertions
+import java.util.*
+
+class Test {
+    @org.junit.jupiter.api.Test
+    internal fun testProblem1_Testcase_EvenNumberWillGetSumOnly() {
+        val intArray: Array<Int> = arrayOf(6,9,24)
+        val expected = 30
+        Assertions.assertEquals(expected, problem1(intArray))
+    }
+
+    @org.junit.jupiter.api.Test
+    internal fun testProblem1_Testcase_OddNumbersOnlyShouldGive0() {
+        val intArray: Array<Int>  = arrayOf(9,1)
+        val expected = 0
+        Assertions.assertEquals(expected, problem1(intArray))
+    }
```

Comment:
Very cool that you have tests, now you can refactor your code without the fear that you will break something :)

Reply:
I have a bad feeling about this...But let's hope for the best :)